qemu/tests/stress_kernel_compile.py

```diff
@@ -20,10 +21,13 @@ def kernelcompile(session, vm_name):
         ip = vm.get_address()
         path = params.get("download_url")
         test.log.info("kernel path = %s", path)
-        get_kernel_cmd = "wget %s" % path
+        get_kernel_cmd = "wget %s --progress=none" % path
+        if session.cmd_status("command -v wget") != 0:
+            inst_wget = utils_package.package_install("wget", session,
```

Comment:
1. It is not must to check the `wget` status,  installation will skip it automatically
2. Need check the status of `utils_package.package_install`, test.error when install failed

Reply:
@yanan-fu , I have upated it ,  please check it. thanks